noetic/distribution.yaml

```diff
@@ -1143,6 +1143,30 @@ repositories:
       version: melodic_dev
     status: maintained
   cob_driver:
+    doc:
+      type: git
+      url: https://github.com/4am-robotics/cob_driver.git
+      version: noetic-release-candidate
+    release:
+      packages:
+      - cob_driver
+      - cob_light
+      - cob_mimic
+      - cob_phidgets
+      - cob_scan_unifier
+      - cob_sick_s300
+      - cob_sound
+      - laser_scan_densifier
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/4am-robotics/cob_driver-release.git
+      version: 0.7.17-1
+    source:
+      type: git
+      url: https://github.com/4am-robotics/cob_driver.git
+      version: noetic-devel
+    status: maintained
+  cob_driver_legacy:
```

Comment:
similar to:
- https://github.com/ros/rosdistro/pull/40670/files#r1570307215
- https://github.com/ros/rosdistro/pull/40671/files#r1570462148
- https://github.com/ros/rosdistro/pull/40672/files#r1570480829

I hope I'm doing the right thing here - could you please confirm that this PR will:
- release a new version `0.7.17` of the following packages from `noetic-release-candidate`
      - cob_driver
      - cob_light
      - cob_mimic
      - cob_phidgets
      - cob_scan_unifier
      - cob_sick_s300
      - cob_sound
      - laser_scan_densifier
- keep the last release `0.7.16` for the following packages available based on `kinetic_release_candidate`
      - cob_base_drive_chain
      - cob_bms_driver
      - cob_canopen_motor
      - cob_elmo_homing
      - cob_generic_can
      - cob_phidget_em_state
      - cob_phidget_power_state
      - cob_relayboard
      - cob_sick_lms1xx
      - cob_undercarriage_ctrl
      - cob_utilities
      - cob_voltage_control